### PR TITLE
PPLUS-1478: We are not able to install spryker-sdk/sdk into the project with PHP 7.4 version

### PR DESCRIPTION
**Pull request by Kyrylo-Meretskyi**
Ticket: [PPLUS-1478](https://spryker.atlassian.net/browse/PPLUS-1478)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,11 +13,11 @@
         "http-interop/http-factory-guzzle": "^1.0",
         "knplabs/github-api": "^3.4",
         "m4tthumphrey/php-gitlab-api": "^11.5",
-        "nikic/php-parser": "4.13.*",
+        "nikic/php-parser": "4.14.*",
         "phpstan/phpstan": "^1.2",
         "spryker-sdk/integrator": "dev-develop",
         "spryker-sdk/sdk-contracts": "^0.2.0",
-        "spryker/code-sniffer": "^0.17.3",
+        "spryker/code-sniffer": "dev-master",
```
Comment (dereuromark) [inline composer.json:20]: Why are those in require and not require-dev?

**Review by alexanderM91 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (alexanderM91) [reply composer.json:20]: Hi @kalaputsik,
could you please check why it's necessary to have code sniffer in the required section?
Thank you!

Comment (kalaputsik) [reply composer.json:20]: Hi @alexanderM91, @Kyrylo-Meretskyi ,
I suppose nowadays we shouldn't require it. 
Previously, when we used the Code Sniffers for the evaluation process it made sense, but now we might move it to the dev

Comment (Kyrylo-Meretskyi) [reply composer.json]: Fixed, spryker/code-sniffer moved to dev dependencies

**Pull request merged**